LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Jean-Philippe Doyle, Copyright (c) 2015
+Copyright (c) 2016
```

Comment:
Ah yeah, got it. Found it funny someone would deliberately remove my name from the copyright!